src/Illuminate/Database/Schema/Grammars/SqlServerGrammar.php

```diff
@@ -566,13 +567,10 @@ protected function typeTimestamp(Fluent $column)
      */
     protected function typeTimestampTz(Fluent $column)
     {
-        if ($column->useCurrent) {
-            $columnType = $column->precision ? "datetimeoffset($column->precision)" : 'datetimeoffset';
+        $columnType = $column->precision ? "datetimeoffset($column->precision)" : 'datetimeoffset';
 
-            return "$columnType default CURRENT_TIMESTAMP";
-        }
-
-        return "datetimeoffset($column->precision)";
```

Comment:
Precision was not checked for existence

To reproduce changes stack:
```console
git log -L 568,577:src/Illuminate/Database/Schema/Grammars/SqlServerGrammar.php
```

* https://github.com/laravel/framework/commit/ecee948b1dec768eca6da66a3285b8f7ef0f400a#diff-92bb93e915885ea74a7ab5271828211cR506
* https://github.com/laravel/framework/commit/15c487fecc00c1a58837ce5692396fec78a5bd9d#diff-92bb93e915885ea74a7ab5271828211cR503
* https://github.com/laravel/framework/commit/5708e11fd9a8a3db29a2f828bdda590ed64bb345#diff-92bb93e915885ea74a7ab5271828211cR533
* https://github.com/laravel/framework/commit/8dd1cd45df52cf115385b3a052f4db5dfc37dc27#diff-92bb93e915885ea74a7ab5271828211cR535  <-- **since this commit `precision` check fails**
* https://github.com/laravel/framework/commit/993b087a9834d5f0f24f774cb7e132dcd1026821#diff-92bb93e915885ea74a7ab5271828211cR559